### ISSUE TITLE
Compile complete list of new and retired content

### ISSUE THREAD

**Issue by charlottehwise** [w3c/wai-website-design#17]
Complete list of new and retired (hidden) content for the new site. Is there anything beyond Shawn's great list from https://github.com/w3c/wai-website-design/issues/5 (included below from ):

https://www.w3.org/WAI/sitemap.html is updated.

Planned new resources:
•more Tutorials
•Inclusion doc draft: http://w3c.github.io/wai-inclusion/ (project page links to requirements, etc: https://www.w3.org/WAI/EO/wiki/Accessibility_and_Inclusion)
•possibly 1-3 more Tips for Getting Started (draft of 3 new ones: http://w3c.github.io/wai-quick-start/)

Planned changes:
• https://www.w3.org/WAI/intro/accessibility.php to be updated and then not need to point to http://www.w3.org/standards/webdesign/accessibility
• Working Group, Interest Group, Task Force, etc. pages (listed under WAI Groups in sitemap) might become out of scope depending on future W3C website decisions.

Changes to IA:
•WCAG 1.0, ATAG 1.0, and UAAG 1.0 resources should have much less predominance. They still need to be find-able, but not primary. 2.0 versions should be primary.

Possible new resources:
•more/better intro guidance on WAI-ARIA
•guidance on specific topics [ask Denis...]

**Comment by shawna-slh** [comment]
There is a huge number of retired pages - probably 1000s.

Per 24 Aug telecon:
1. There are at least 3 "levels":
1.a. in main nav - easily findable
1.b. linked somewhere, but doesn't need to be in main nav and highly findable
1.c. old content that shouldn't be linked anywhere
2. I'm happy to annotate any auto-generated list of pages.

**Comment by charlottehwise** [comment]
Changes I am making to the IA:

- I have folded the Accessiblity, Usability, and Inclusion doc draft into Understanding Accessibility. This content is great!
- I have added an Archive link to the main site footer where items which need to be findable via search, etc. can be included but which no longer need a space in primary or secondary nav.
- I have also deprecated the WCAG 1.0, ATAG 1.0, UAAG 1.0 resource-related pages from their status in the secondary navigation. They can go to the archive, too.
-Added the IndieUI material back into the sitemap (under Guidelines for Measuring Accessibility)
- I will leave the WAI Working Groups and Interest Groups in under About WAI for now, but that could also be deprecated depending on future W3C website decisions.
